packages/as-proto-gen/src/generate/field.ts

```diff
@@ -105,7 +105,7 @@ export function generateFieldDecodeInstruction(
     if (isRepeated) {
       return `
         case ${fieldNumber}:
-          if ((tag & 7) === 2) {
+          if ((tag & 7) === 2 && tag !== 26) {
```

Comment:
Where is the value `26` coming from?

Reply:
26 is array bytes

Reply:
But where is the actual list of these numbers?